Overview.md

```diff
@@ -0,0 +1,71 @@
+## Overview
+
+### HomeID
+
+ZWave has the notion of a `HomeID` which is a randomly generated ID
+assigned to the dongle. It is used for defining your network of devices.
+All of the devices which are paired to a dongle will only talk to a
+dongle with the HomeID it was paired with.
+
+Factory resetting the dongle will assign a new randomly generated HomeID.
+
+### NodeI
+
+NodeIDs are in the range 1-255 and refers to the particular device that
+you're dealing with. NodeId 1 is assigned to the dongle and remaining
+devices get assigned an incrementing ID each time a device is paired.
+Unpairing and repairing a device will give it a new NodeID.
+
+The HomeId and NodeId are combined together to create the unique device
+id. The device id will be zwave-homeid-NodeId
+
+### ValueIDs
+
+A zwave ValueID is made up of 4 components:
+
+* NodeID - (1-255) Refers to the device that you're dealing with. NodeId 1 is assigned to the dongle and remaining devices get assigned an incrementing ID each time a device is paired. Unpairing and repairing a device will give it a new NodeID.
+* Class ID - (aka COMMAND_CLASS). This describe the major functionality that the command class offers.
+* Instance - normally 1, but may have other values if the device supports multiple instances (like a device with 2 outlets where each outlet command be controlled)
+* Index - defines 1 particular attribute of the command class. The file: https://github.com/mozilla-iot/open-zwave/blob/master/cpp/src/ValueIDIndexesDefines.def is what OpenZWave uses.
+
+### OpenZWave
+
+We use a fork of OpenZWave https://github.com/mozilla-iot/open-zwave/ and use
```

Comment:
Does our fork have changes, or are we just using it to pin a specific version?

Reply:
Currently, it's just a tag to pin a specific version, but in the past I've had changes (usually backporting a fix that would cause a crash without the fix).